zerver/webhooks/github/view.py

```diff
@@ -674,6 +683,13 @@ def get_topic_based_on_type(payload: WildValue, event: str) -> str:
             title=payload["pull_request"]["title"].tame(check_string),
         )
     elif event.startswith("issue"):
+        if event == "issue_comment" and is_pull_request_comment_event(payload):
+            return TOPIC_WITH_PR_OR_ISSUE_INFO_TEMPLATE.format(
+                repo=get_repository_name(payload),
+                type="PR",
```

Comment:
Why are we duplicating the whole format statement from below? Is it not sufficient to just set the `type`?

```python
type="PR" if is_pull_request_comment_event(payload) and event == "issue_comment" else "issue"
```

I think the `event == "issue_comment"` condition can be dropped too, in case there are other events where the payload is a pull request payload, but the event name starts with "issue".

Reply:
Updated for this feedback.